frontend/javascripts/oxalis/view/layouting/tracing_layout_view.tsx

```diff
@@ -113,10 +113,6 @@ class TracingLayoutView extends React.PureComponent<PropsWithRouter, State> {
     };
   }
 
-  componentDidMount() {
-    window.addEventListener("resize", this.debouncedOnLayoutChange);
```

Comment:
I moved this to the `setControllerStatus` code so that the handler is not triggered when wk is still loading (in that case, no dataset layer exist and the valid zoom range cannot be derived).

Alternatively, one could return a nullable zoom range (or a dummy one), but I think it's cleaner to just don't do anything that relies on the dataset layers when these are not fully loaded yet.